src/sidebar.jsx

```diff
@@ -149,6 +149,12 @@ const DropdownWithKebab = ({ selected, path, setPath, showHidden, setShowHidden,
                         {_("Create directory")}
                     </DropdownItem>
                 </>}
+                <DropdownItem
+                  id="create-link" key="create-link"
+                  onClick={() => { createLink(Dialogs, "/" + path.join("/") + "/", files) }}
```

Comment:
Can you create some constant variable for the currentDirectory = "/" + path.join("/") + "/"

As you reuse it already.